src/components/Stepper/Step/Illustration.js

```diff
@@ -35,7 +36,7 @@ function Illustration({ status, index }) {
             color: ${theme.positiveContent};
           `}
         >
-          <IconBlock size="large" />
+          <img src={blockIcon} height={48} width={48} />
```

Comment:
just to double check, did we check that this change is not making the other assets display in a weird way?